dcicutils/http_utils.py

```diff
@@ -0,0 +1,31 @@
+from contextlib import contextmanager
+import requests
+from typing import Callable, Optional
+from dcicutils.tmpfile_utils import temporary_file
+
+
+@contextmanager
```

Comment:
May want to provide a non-download version as well

Reply:
Okay all set, thanks.